docs/openapi/components/schemas/common/TransformEvent.yml

```diff
@@ -0,0 +1,203 @@
+$linkedData:
+  term: TransformEvent
+  '@id': https://w3id.org/traceability#TransformEvent
+title: Transform Event
+description: A product transformation event, extending the Event schema.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+      - type: string
+        enum:
+          - Event
+          - TransformEvent
+  eventType:
```

Comment:
Am I interpreting this correctly that these are all types of steel transformations?

Reply:
this transform event should be a generic one for different commodities, not steel specific. enums for eventType have be removed.